tests/Makefile

```diff
@@ -8,7 +8,7 @@ integration: verify-vendor test
 
 .PHONY: test
 test: ## Run only the tests.
```

Comment:
Can we have the old version of this as a different target?  Say `test-legacy` or `test-upstream` or some such?  When this breaks, I'd like to get our repo mostly unstuck quickly.